src/renderer/components/select/select.test.tsx

```diff
@@ -0,0 +1,137 @@
+    di.override(rendererExtensionsInjectable, () => computed(() => [] as LensRendererExtension[]));
+
+    UserStore.createInstance();
+    ThemeStore.createInstance();
+  });
+
+  afterEach(() => {
+    ThemeStore.resetInstance();
+    UserStore.resetInstance();
+
+    mockFs.restore();
+  });
+
+  it("should render the select", async () => {
+    const options = [
+      {
+        label: "Option one label",
+        value: "optionOneValue",
+      },
+      {
+        label: "Option two label",
+        value: "optionTwoValue",
+      },
+    ];
+
+    const onChange = jest.fn();
+
+    const { container } = render(<Select onChange={onChange} options={options} />);
+
+    expect(container).toBeInstanceOf(HTMLElement);
+  });
+
+  it("should show selected option", async () => {
+    const options = [
+      {
+        label: "Option one label",
+        value: "optionOneValue",
+      },
+      {
+        label: "Option two label",
+        value: "optionTwoValue",
+      },
+    ];
+
+    const onChange = jest.fn();
+
+    const { container } = render(<Select value={options[0].value} onChange={onChange} options={options} />);
+    const selectedValueContainer = container.querySelector(".Select__single-value");
+
+    expect(selectedValueContainer.textContent).toBe(options[0].label);
+  });
+
+  it("should reflect to change value", async () => {
+    const options = [
+      {
+        label: "Option one label",
+        value: "optionOneValue",
+      },
+      {
+        label: "Option two label",
+        value: "optionTwoValue",
+      },
+    ];
+
+    const onChange = jest.fn();
+
+    const { container, rerender } = render(<Select value={options[0].value} onChange={onChange} options={options} />);
+    const selectedValueContainer = container.querySelector(".Select__single-value");
+
+    expect(selectedValueContainer.textContent).toBe(options[0].label);
+
+    rerender(<Select value={options[1].value} onChange={onChange} options={options} />);
+
+    expect(selectedValueContainer.textContent).toBe(options[1].label);
+
+    rerender(<Select value={null} onChange={onChange} options={options} />);
+
+    expect(selectedValueContainer).not.toBeInTheDocument();
+  });
+
+  it("should unselect value if no value is passed", async () => {
+    const options = [
+      {
+        label: "Option one label",
+        value: "optionOneValue",
+      },
+      {
+        label: "Option two label",
+        value: "optionTwoValue",
+      },
+    ];
+
+    const onChange = jest.fn();
+
+    const { container, rerender } = render(<Select value={options[0].value} onChange={onChange} options={options} />);
+    const selectedValueContainer = container.querySelector(".Select__single-value");
+
+    expect(selectedValueContainer.textContent).toBe(options[0].label);
+
+    rerender(<Select value={null} onChange={onChange} options={options} />);
+
+    expect(selectedValueContainer).not.toBeInTheDocument();
```

Comment:
nit, I would explicitly call `querySelector` again here because it might be that a different container was produced.